src/Moryx.Resources.Management/ModuleController/ModuleConsole.cs

```diff
@@ -31,12 +31,6 @@ internal class ModuleConsole : IServerModuleConsole
 
         #endregion
 
-        #region Fields
-
-        private IResourceInitializer[] _initializers;
```

Comment:
what was that for?

Reply:
@Toxantron, maybe you can answer that from the top of your head. But it was a private field that was not used in the class, that's why I removed it 😊